### PR TITLE
HH/HI content changes

### PR DESCRIPTION
**Pull request by MebinAbraham**
### What is the context of this PR?

Few content changes for ENG/WLS HH/Hi.

### How to review

Ensure content changes are as outlined on google sheet attached to the Trello card.

### Checklist

- [x] Jsonnet files conform to the latest [style guide](/ONSdigital/eq-questionnaire-schemas/blob/master/style_guide.md)

#### England

- [Household](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=H&region_code=GB-ENG&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/hh-hi-content-changes/schemas/en/census_household_gb_eng.json)

- [Individual](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-ENG&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/hh-hi-content-changes/schemas/en/census_individual_gb_eng.json)

### CODE REVIEW

File: source/jsonnet/england-wales/individual/blocks/identity-and-health/ethnic_group_asian_other.jsonnet [hunk 1]
```diff
@@ -20,25 +29,18 @@ local question(title, region_code) = (
...
-    placeholders.personName(),
-  ],
-};
 
-function(region_code) {
+{
   type: 'Question',
   id: 'other-asian-or-asian-british-ethnic-group',
-  page_title: 'Other Asian or Asian British ethnic group or background',
+  page_title: 'Other Asian ethnic group or background',
```
Comment (JamesGardiner) [inline source/jsonnet/england-wales/individual/blocks/identity-and-health/ethnic_group_asian_other.jsonnet:36]: I don't see this change specced on the relevant tab of the google doc?

Comment (MebinAbraham) [reply source/jsonnet/england-wales/individual/blocks/identity-and-health/ethnic_group_asian_other.jsonnet:36]: It was something that was missed. Ill add it to the docs.

**Review by JamesGardiner — COMMENTED** [review]
LGTM apart from the page title change - is this specified anywhere?

**Review by JamesGardiner — APPROVED** [review]

**Review by LJBabbage — APPROVED** [review]
LGTM, checked the content as well

**Review by petechd — APPROVED** [review]
👍

**Pull request merged**